concurrent/futures/thread.py

```diff
@@ -109,6 +110,7 @@ def __init__(self, max_workers=None):
         self._threads = set()
         self._shutdown = False
         self._shutdown_lock = threading.Lock()
+        self._thread_name_prefix = thread_name_prefix
```

Comment:
Your PR seems incomplete. This part of the code differs from upstream: https://github.com/python/cpython/blob/master/Lib/concurrent/futures/thread.py#L109-L110